apps/hash-agents/app/agents/generate-text-from-prompt/__main__.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+import structlog.stdlib
+from langchain.chat_models import ChatOpenAI
+from langchain.schema import HumanMessage, SystemMessage
+
+from .io_types import Input, Output
+
+logger = structlog.stdlib.get_logger(__name__)
+
+SYSTEM_MESSAGE_CONTENT = (
+    "You are a text generator. Given a topic or a question, generate a"
+    "paragraph of text accurately describing the given topic or answering the"
+    " question. You must be concise. Do not comment on controversial issues"
+    " including politics."
+)
+
+def main(agent_input: Input) -> Output:
+    chat = ChatOpenAI(model_name="gpt-3.5-turbo", temperature=0)
```

Comment:
Did we experiment with the temperature here? Cold!

Reply:
No – @Alfred-Mountfield do you think I should do something different?

Reply:
I'd recommend we stick with 0 until there are reasons to increase it. Reasons I can think of (for the future):
- When it's a part of a chain and we are trying to retry things that aren't satisfactory (we want a bit of randomness then to be able to regenerate a broken response and hope it's not broken)
- When a user wants more random output (e.g. generating text, they may want multiple different suggestions they can pick from)

For the _current_ purposes of the branch and these PRs, I'd recommend we keep it low so it's (mostly) deterministic to avoid unexpected behaviour for a given set of working prompts right now.